drivers/flash/flash_ospi_is25wx.c

```diff
@@ -823,6 +826,10 @@ static int flash_alif_ospi_init(const struct device *dev)
 
 	dev_data->ISSI_Flags |= FLASH_POWER;
 
+#ifdef CONFIG_ALIF_OSPI_FLASH_XIP
```

Comment:
Please USE if(IS_ENABLED()) #ifdef are not nice. We should avoid these

Reply:
https://docs.zephyrproject.org/apidoc/latest/group__sys-util.html#ga111fe4e9d63758262fc6810a782cb32a
Check this link.